deploy/helm/elastic-agent/templates/integrations/_auto_ops/_auto_ops.tpl

```diff
@@ -0,0 +1,125 @@
+{{- define "elasticagent.autoops.init" -}}
```

Comment:
please add the autoops team as the owner of this folder `deploy/helm/elastic-agent/templates/integrations/_auto_ops` inside .github/CODEOWNERS